Update project-board-visibility.md (#10129)
github/docs: data/reusables/project-management/project-board-visibility.md

> By default, your project board is private and only visible to people with read, write, or admin permissions to the project board. {% ifversion ghae %}An internal{% else %}A public{% endif %} project board is visible to {% ifversion ghae %}any enterprise members{% else %}anyone{% endif %} with the project board's URL. Note that this only applies to project boards owned by a organization or user account. Project boards associated with a repository will share the visibility of their repository, i.e. a private repository will have a private board, and this setting cannot be changed.

Reviewer: I think it might be worth making a slightly larger change here:

```suggestion
By default, user-owned and organization-wide project boards are private and only visible to people with read, write, or admin permissions to the project board. {% ifversion ghae %}An internal{% else %}A public{% endif %} project board is visible to {% ifversion ghae %}any enterprise members{% else %}anyone{% endif %} with the project board's URL. Repository-level project boards share the visibility of their repository. That is, a private repository will have a private board, and this visibility cannot be changed.
```
